app/views/thesis/_duplicate_thesis.html.erb

```diff
@@ -0,0 +1,15 @@
+<tr>
+  <td><%= link_to duplicate_thesis.title, thesis_process_path(duplicate_thesis.id) %></td>
```

Comment:
`link_to title_helper(duplicate_thesis), thesis_process_path(duplicate_thesis.id)` would be preferred to use the existing helper method to handle theses without real titles.

Reply:
Oh yeah, I forgot about that helper - will add that. Do you want to re-review with that, or can I squash/rebase as part of adding that?